data/etcd/store.go

```diff
@@ -0,0 +1,171 @@
+// Packaged etcd was influenced by and borrows helper functions from:
+// https://github.com/kubernetes/kubernetes/pkg/storage/etcd3
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcd
+
+import (
+	"context"
```

Comment:
this import doesn't compile on my environment
I think we need to use "golang.org/x/net/context"